README.md

```diff
@@ -1,2 +1,54 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
 # Review-past-projects
-this project about code review. I select previous project to give review on it.
+
+> This is previous project about code review.
+
+
+## Built With
+
+- HTML 
+- CSS
+
+## Live Demo (if available)
+
+[Live Demo Link]()
+
+
+## Getting Started
+git clone: https://github.com/RashidMenhas/Review-past-projects.git
+
+cd Review-past-projects
+
+
+
+
+
```

Comment:
- please remove ❌ blank line from your project. "because it is a bad coding practice" 🙅